backend/lib/spree/backend/tabs/product/images.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+require 'spree/backend/tabs/product/base'
+
+module Spree
+  module Backend
+    module Tabs
+      class Product
+        class Images < Base
+          def name
+            'Images'
+          end
+
+          def presentation
+            view_context.t('spree.images')
```

Comment:
There is no need to use the `view_context`. `I18n.t('spree.images')` should work as well.